src/IRremoteESP8266.h

```diff
@@ -614,6 +614,13 @@
 #define SEND_DOSHISHA         _IR_ENABLE_DEFAULT_
 #endif  // SEND_DOSHISHA
 
+#ifndef DECODE_MULTIBRACKETS
+#define DECODE_MULTIBRACKETS _IR_ENABLE_DEFAULT_
+#endif  // DECODE_MULTIBRACKETS
+#ifndef SEND_MULTIBRACKETS
+#define SEND_MULTIBRACKETS   _IR_ENABLE_DEFAULT_
```

Comment:
Do we want to try and line up `_IR_ENABLE_DEFAULT_` ? if so what is the "best style" ?

Reply:
It's a "meh" issue IMHO. I only line them up within a protocol. Or when it's convenient.
e.g. I'm not going to re-do every entry when I add a longer protocol name.